hw2/03-count.js

```diff
@@ -0,0 +1,22 @@
+const input = document.getElementById("input");
+const textContainer = document.getElementById("text-container");
+
+input.addEventListener("input", () => {
+  const keyword = input.value;
+  const text = textContainer.textContent;
+  const words = text.split(" ");
+
+  if (keyword.trim() !== "") {
+    const highlightedText = words
+      .map((word) =>
+        word.toLowerCase().includes(keyword.toLowerCase())
+          ? `<span class="bg-warning">${word}</span>`
```

Comment:
this is fine, but I think it's better to use `createElement` and add to `classList` instead since it's harder to make mistake and typo that way